akka-persistence-typed/src/test/scala/akka/persistence/typed/scaladsl/PersistentBehaviorSpec.scala

```diff
@@ -132,25 +158,29 @@ object PersistentActorSpec {
             .andThen {
               loggingActor ! firstLogging
             }
+
         case LogThenStop ⇒
-          Effect.none.andThen {
-            loggingActor ! firstLogging
-          }.andThenStop
+          Effect.none
+            .andThen {
+              loggingActor ! firstLogging
+            }
+            .andThenStop
```

Comment:
wouldn't `{ loggingActor ! firstLogging; Effect.stop }` make more sense? There shouldn't be a (real) difference right?

Reply:
That's API change, it was never able to return effects, since we would not want that to return another Persist I believe

Reply:
I'm not sure I follow? Wouldn't the implementation `{ loggingActor ! firstLogging; Effect.stop }` be slightly nicer and otherwise equivalent to `Effect.none.andThen { loggingActor ! firstLogging }.andThenStop`?

Reply:
I'm confused, are you arguing to remove the `andThenStop`? That's not something I added. I existed, and I don't want to get into changing the persistent behavior API in this PR which is already so all over the place that it's difficult to get concrete feedback